### DISCUSSION TITLE
can anyone help me

### DISCUSSION THREAD

**Discussion by ebhaskarreddy** [learndevopsonline/learndevopsonline#1591] [Q&A]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/149461818/0f140ca7-45a3-49db-aa46-c81c836a920d)

**Comment by sreedharm07** [comment]
Hi @ebhaskarreddy 
expence .yml file should not be inside roles directory 
place expence.yml file in expence-ansible
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/8817f934-2eeb-45b9-a6c5-cde0c994ab54)

**Reply by ebhaskarreddy** [reply]
yes got now thanks
